### ISSUE TITLE
 The upsample_bicubic2d operator implemented by the CPU, when the input and output shapes are the same, There is a problem of data loss when copying data

### ISSUE THREAD

**Issue by huaxialongguang** [pytorch/pytorch#49159]
## 🐛 Bug

The operator upsample_bicubic2d  implemented by the CPU, when the input and output shapes are the same, There is a problem of data loss when copying data.
pytorch1.5 pytorch/ aten / src / ATen / native / UpSampleBicubic2d.cpp，In line 30, channels should be  changed to channels*nbatch
## To Reproduce

1.code sample：
```
import torch
x = torch.tensor([[[[2., 4.],[6., 4.]],[[5., 7.], [5., 2.]]], [[[9., 1.],[6., 7.]],[[9., 2.],[9., 4.]]]])
y = torch.nn.functional.interpolate(x, mode = 'bicubic', scale_factor =1)
print(y)
```
2.CPU Result
```
tensor([[[[2., 4.],[6., 4.]],[[5., 7.],[5., 2.]]],[[[0., 0.],[0., 0.]], [[0., 0.],[0., 0.]]]])
```
3.GPU  Result
```
tensor([[[[2., 4.], [6., 4.]],[[5., 7.], [5., 2.]]],[[[9., 1.],[6., 7.]],[[9., 2.],[9., 4.]]]], device='cuda:0')
```

## Expected behavior

pytorch/ aten / src / ATen / native / UpSampleBicubic2d.cpp，In line 30, channels should be  changed to channels*nbatch
## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

cc @ezyang @gchanan @zou3519 @bdhirsh @VitalyFedyunin @albanD @mruberry

**Comment by mruberry** [comment]
Thank you for reporting this issue, @huaxialongguang. We'll take a look.

**Comment by zou3519** [comment]
I can repro this on PyTorch 1.7

**Comment by ngimel** [comment]
@huaxialongguang thank you for your investigation, can you please submit a PR with your proposed fix?
